node/node.go

```diff
@@ -64,6 +75,13 @@ type NodeProvider interface {
 // underlying options, the last NodeControllerOpt will win.
 func NewNodeController(p NodeProvider, node *corev1.Node, nodes v1.NodeInterface, opts ...NodeControllerOpt) (*NodeController, error) {
 	n := &NodeController{p: p, n: node, nodes: nodes, chReady: make(chan struct{})}
+
+	// Apply instance type label.
+	labels := make(map[string]string)
```

Comment:
Can we move this to the default generated node config?

Reply:
Makes sense, will do!